plugins/keybindings/msd-keybindings-manager.c

```diff
@@ -549,9 +550,11 @@ msd_keybindings_manager_start (MsdKeybindingsManager *manager,
 
         for (i = 0; i < screen_num; i++) {
                 screen = gdk_display_get_screen (dpy, i);
-                gdk_window_add_filter (gdk_screen_get_root_window (screen),
+                window = gdk_screen_get_root_window(screen);
+                gdk_window_add_filter (window,
                                        (GdkFilterFunc) keybindings_filter,
                                        manager);
+                XSelectInput(GDK_DISPLAY_XDISPLAY(dpy), GDK_WINDOW_XID(window), KeyPressMask);
```

Comment:
Ideally this would be guarded with `gdk_error_trap_push()` and `gdk_error_trap_pop_ignored()`, no?